cmd/remove.go

```diff
@@ -47,8 +50,50 @@ use 'hosts disable' instead.
 	},
 }
 
+// removeDomainsCmd represents the remove command
+var removeDomainsCmd = &cobra.Command{
+	Use:   "domains",
+	Short: "Remove domains from your hosts file.",
+	Long: `
+Completely remove domains from your hosts file.
+It cannot be undone unless you have a backup and restore it.
+`,
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		profile, _ := cmd.Flags().GetString("profile")
+		all, _ := cmd.Flags().GetBool("all")
+
+		if !all && profile == "" {
+			return host.MissingProfileError
+		}
+
+		if profile == "default" {
+			return host.DefaultProfileError
+		}
+		return nil
+	},
+	RunE: func(cmd *cobra.Command, args []string) error {
+		profile, _ := cmd.Flags().GetString("profile")
+		dst, _ := cmd.Flags().GetString("host-file")
+
+		err := host.RemoveDomains(&host.RemoveDomainsOptions{
```

Comment:
Same comment as `RemoveProfile` above

```suggestion
		err := host.RemoveDomains(dst, profile, domains)
```